bottles/backend/wine/winecommand.py

```diff
@@ -107,6 +107,10 @@ def __init__(
         self.arguments = arguments
         self.cwd = self._get_cwd(cwd)
         self.runner, self.runner_runtime = self._get_runner_info()
+        self.gamescope_activated = (
+            environment["GAMESCOPE"] == "1" if "GAMESCOPE" in environment
+            else self.config.Parameters.gamescope
+        )
```

Comment:
I found this to be the simplest way to check (though not sure if the most orthodox one).

Reply:
Looks good